server/src/routers/userRouter.js

```diff
@@ -1,21 +1,31 @@
-const express = require('express')
-const path = require('path')
+const express = require('express');
 const router = new express.Router()
 
-const User = require('../models/users')
+const User = require("../models/users")
```

Comment:
It seems you missed converting back to single quotes here. Here comes the need for the linter. It would make our lives so much easier.

Reply:
Will be taken care of in the next commit.